RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Add support for python >= 3.6
```

Comment:
I think the description would be better as:

> Change minimum Python version from 3.7 to 3.6

Reply:
yup!